src/Modules/Authontication/Components/Register/Register.tsx

```diff
@@ -1,9 +1,271 @@
+    toast.error(
+     error?.response?.data?.message || "Register unsuccessful. Please try again"
+   );
+    console.log(error);
+   }
+ }
+
+
+ 
+ const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+ const toggleVisibility = (setterFunction: any) => {
+   setterFunction((prevState: any) => !prevState);
+ };
+  
   return (
-  <h1 className='text-white'>Register</h1>
+    <>
+      <div className="auth-title my-4">
+    <p className="text-white">Welcome to PMS</p>
+    <h3 className="main-color title">
+          <span className="frist-ch position-relative">C</span>reate New Account
+        </h3>
+        <div className="d-flex justify-content-center">
+        <img className='' src={regImg}/>
+        </div>
+  </div>
+
+  <form className='container-form' onSubmit={handleSubmit(onSubmit)}>
+    <div className="row">
+      <div className="col-md-6">
+      <div className=" ">
+    <label className="main-color my-1">Email</label>
+      <div className="input-group ">
+      <input type="text" className="form-control form-input"
+       placeholder="Enter your email"
+       aria-label="email" aria-describedby="basic-addon1"
+       {...register("email",EmailValidation)}
+       />
+      </div>
+     
+    </div>
+    {errors.email && (
+            <span className="text-danger">
+              {String(errors.email.message)}
+            </span>
+    )}
+
+      </div>
+
+      <div className='col-md-6'>
+      <div className="">
+    <label className="main-color my-1">userName</label>
+      <div className="input-group ">
+      <input type="text" className="form-control form-input"
+       placeholder="Enter your userName "
+       aria-label="userName" aria-describedby="basic-addon1"
+       {...register("userName",FieldValidation)}
+       />
+      </div>
+     
+    </div>
+    {errors.userName && (
+            <span className="text-danger">
+              {String(errors.userName.message)}
+            </span>
+    )}
+
+      </div>
+
+    </div>
     
+
+    <div className="row">
+      <div className="col-md-6">
+      <div className="">
+    <label className="main-color my-1">Country </label>
+      <div className="input-group ">
+      <input type="text" className="form-control form-input"
+       placeholder="Enter your country "
+       aria-label="country" aria-describedby="basic-addon1"
+       {...register("country",FieldValidation)}
+       />
+      </div>
+     
+    </div>
+    {errors.country && (
+            <span className="text-danger">
+              {String(errors.country.message)}
+            </span>
+    )}
+
+      </div>
+      <div className='col-md-6'>
+<div className="">
+    <label className="main-color my-1">phoneNumber </label>
+      <div className="input-group ">
+      <input type="text" className="form-control form-input"
+       placeholder="Enter your phoneNumber "
+       aria-label="phoneNumber" aria-describedby="basic-addon1"
+       {...register("phoneNumber",FieldValidation)}
```

Comment:
We can add a phone validation logic to prevent adding normal charchters.
The type of the input should also be `tel`.

Reply:
OK , DONE